src/gui/processing/qgsprocessingmodelerparameterwidget.h

```diff
@@ -82,6 +84,19 @@ class GUI_EXPORT QgsProcessingModelerParameterWidget : public QWidget
 
     ~QgsProcessingModelerParameterWidget() override;
 
+    /**
+     * Sets the \a context in which the modeler parameter widget is shown, e.g., the
+     * parent model algorithm and other relevant information which allows the widget
+     * to fine-tune its behavior.
+     */
+    void setWidgetContext( const QgsProcessingParameterWidgetContext &context );
+
+    /**
+     * Register a Processing context \a generator class that will be used to retrieve
```

Comment:
Register**s**

Reply:
@pblottiere I saw your comments on other PRs as well, I've always though we were using an imperative style in code documentation. I didn't check if it's in the coding guidelines, but, either way we should put it there.

Reply:
We try to follow Qt documentation style, which is "gets the ....", "sets the ..."

Reply:
Added: https://github.com/qgis/QGIS-Website/pull/567

Reply:
Thanks @elpaso!